src/Android/Avalonia.Android/Platform/SkiaPlatform/InvalidationAwareSurfaceView.cs

```diff
@@ -13,7 +13,6 @@ public abstract class InvalidationAwareSurfaceView : SurfaceView, ISurfaceHolder
         bool _invalidateQueued;
         readonly object _lock = new object();
         private readonly Handler _handler;
-        
```

Comment:
nit

Reply:
Unclear... Do you mean two empty lines needed after fields declaration? :)